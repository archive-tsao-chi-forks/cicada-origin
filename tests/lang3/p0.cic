_ : String
_ = "abc"

_ : { x : String }
_ = { x = "abc" }

_ : { x : { x : String } }
_ = { x = { x = "abc" } }

_ : { x : { x : String } }
_ = { x = { x = "abc" } y = "123" }

z : { x : { x : String } y : String }
z = { x = { x = "abc" } y = "123" }

@show z.x
@show z.y
