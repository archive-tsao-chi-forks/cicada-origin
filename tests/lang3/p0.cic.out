{
  x = {
    x = "abc"
  }
  y = "123"
}: {
  x : {
    x : String
  }
  y : String
}
{
  x = "abc"
}: {
  x : String
}
"123": String
